### PR TITLE
Update PREREQS.md

### PR DESCRIPTION
**Pull request by ZMASLO**
DXVK and proton wasn't working on my Radeon R9 290 and I figured out why. I provided solution for problem (many people had it) and also I provided verification that Vulkan is working by running command vulkaninfo.

I hope that I helped.

### CODE REVIEW

File: PREREQS.md [hunk 1]
```diff
@@ -28,6 +28,22 @@ sudo apt install mesa-vulkan-drivers mesa-vulkan-drivers:i386
 
 Provide your user password when requested and reboot after the last command completes to ensure the driver has updated correctly.
 
+On Radeon R9 200/300 series you have to blacklist radeon module and add following parameters to amdgpu:
+
+```
+echo "blacklist radeon" | sudo tee --append /etc/modprobe.d/blacklist.conf
```
Comment (nanonyme) [inline PREREQS.md:34]: Might want to have a warning here. It's not enabled by default because it's not done even with latest kernel releases

**Comment by kisak-valve** [comment]
Hello @ZMASLO, this has moved to https://github.com/ValveSoftware/Proton/wiki/Requirements.

**Comment by codeman101** [comment]
Why are the instructions in the PREREQS.md only for ubuntu users? What about Fedora, Arch and OpenSuse?

**Comment by omicron-b** [comment]
@codeman101 from https://store.steampowered.com/about/ 
```
Supported Operating Systems

Windows 7, 8, 8.1, 10, or later.
Intel Mac, OS X version 10.11 (El Capitan), or later.
Ubuntu Linux (latest version).
```

I am not saying that the instructions should not be added by the community, but that is probably why they were not added by Valve.

**Comment by aeikum** [comment]
This stuff should go into the wiki now.

**Pull request closed without merging**